cmip6_downscaling/workflows/utils.py

```diff
@@ -418,7 +419,7 @@ def regrid_dataset(
         ds_rechunked = rechunk_zarr_array_with_caching(
             zarr_array=ds, chunking_approach='full_time', max_mem='1GB'
         )
-
+    time.sleep(1)
```

Comment:
copy that same "we're going to sleep here for 1 second because...." comment here and above the change below!